### PR TITLE
Unit tests: consistent use of Token units versus TokenWei units

### PR DESCRIPTION
**Pull request by abhayks1**
PR for consistent use of Token units versus TokenWei units: review and correct unit tests for correct naming and use of Token vs TokenWei (ST vs STWei, BT vs BTWei, UT vs UTWei)

### CODE REVIEW

**Comment by abhayks1** [comment]
#81

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ experience to build mainstream applications on Ethereum.
 
 Detailed changelog:
 
+- Unit tests: Consistent use of Token units versus TokenWei units ([openst-protocol#1](https://github.com/OpenSTFoundation/openst-protocol/pull/82))
```
Comment (benjaminbollen) [inline CHANGELOG.md]: can already be under new header v0.9.2

File: test/BrandedToken.js [hunk 2]
```diff
@@ -97,7 +97,8 @@ contract('BrandedToken', function(accounts) {
 		})
 
 		it('fails to burn by openSTProtocol if msg.value != 0', async () => {
-            await Utils.expectThrow(token.burn(beneficiary, ST1, { from: openSTProtocol, value: 1 }));
+						const amountBT = new BigNumber(web3.toWei(1, "ether"));
```
Comment (benjaminbollen) [inline test/BrandedToken.js]: indentation

File: test/OpenSTUtility.js [hunk 3]
```diff
@@ -300,83 +303,86 @@ contract('OpenSTUtility', function(accounts) {
...
+			var redeemReturns = await openSTUtility.redeemSTPrime.call(redeemSTP.toNumber(), { from: redeemer, value: 2 });
 
-            // call block number is one less than send block number
-            unlockHeight = redeemReturns[1].plus(1)
-            var checkRedemptionIntentHash = await openSTUtility.hashRedemptionIntent.call(uuidSTPrime, redeemer, 2, 2, unlockHeight);
-            result = await openSTUtility.redeemSTPrime(2, { from: redeemer, value: 2 });
+      // call block number is one less than send block number
+      unlockHeight = redeemReturns[1].plus(1)
+      var checkRedemptionIntentHash = await openSTUtility.hashRedemptionIntent.call(uuidSTPrime, redeemer, 2, redeemSTP, unlockHeight);
+      result = await openSTUtility.redeemSTPrime(redeemSTP.toNumber(), { from: redeemer, value: redeemSTP });
```
Comment (benjaminbollen) [inline test/OpenSTUtility.js]: on `redeemSTP.toNumber()` why convert it to a Number? (similarly on L 377, L385)

Comment (abhayks1) [reply test/OpenSTUtility.js]: Although it works but you are right it's not required. 
toNumber() is required while comparing values while asserting.

File: CHANGELOG.md (hunk 1 above)
Comment (abhayks1) [reply CHANGELOG.md]: Changes Done

**Pull request merged**